javascript/chronometer.js

```diff
@@ -1,26 +1,35 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = 0
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(() =>
+      this.currentTime++, 1000)
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
   getSeconds() {
-    // ... your code goes here
```

Comment:
Good work Nina! You can make your solution even easier if you use a modulo operator `%` if you're unfamiliar with it, check out [this link](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math#Arithmetic_operators)
```
return this.currentTime % 60;

Reply:
I tried to implement this but it's not working so well now with the milliseconds. I guess this.currentTime%6000 is not the answer, right?